modules/ROOT/pages/building-an-https-service.adoc

```diff
@@ -9,110 +9,135 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", ti
+CloudHub supports deploying HTTPS- and SSL-based services so you ensure your data confidentiality. 
+These configuration steps shown are meant for deploying your Mule app to CloudHub.
 
-If you are using Mule 4, you can follow these steps, or take advantage of API Manager integration with secrets manager. 
-Follow the xref:api-manager::building-https-proxy.adoc[Building an HTTPS API Proxy] instructions for more information.
+If you are using:
 
-If you intend to deploy your app locally, see xref:3.8@mule-runtime::tls-configuration.adoc[TLS Configuration]. 
-The xref:deployment-strategies.adoc[Deployment Strategies] documentation offers a closer look at how the configuration should differ between each deployment scenario.
+* *Mule 3 with Anypoint Studio 6*
++
+Follow these steps to build an HTTPS Proxy.
+* *Mule 4 with Anypoint Studio 7*
++
+Take advantage of Anypoint API Manager integration with the secrets manager. 
++
+See xref:api-manager::building-https-proxy.adoc[Building an HTTPS API Proxy in Mule 4].
+
+If you plan to deploy your app locally, see xref:3.8@mule-runtime::tls-configuration.adoc[TLS Configuration]. 
+
+== Prerequisites 
 
-Before you begin, build a service and deploy it. It can be a simple Hello World service.
+Before you begin, build a service (such as a simple "Hello World" service) and deploy it.
 
-You can now modify your existing service to use HTTPS as detailed in the following steps.
+== Modify Your Service to Use HTTPS
 
-. Generate a keystore if you don't have one already.
- You can do this using the command line and the keytool utility from the JDK.
+For detailed information about the Anypoint Connector for HTTP, see the xref:3.8@mule-runtime::http-connector.adoc[HTTP Connector for Mule 3] documentation.
+
+To modify your existing service to use HTTPS:
+
+. Generate a `keystore` file.
 +
-[IMPORTANT]
---
-To generate a self signed certificate now you must specify the hostname in your command, the property 'SAN=DNS:localhost,IP:127.0.0.1' is key in the example below
---
+You can create a `keystore` file using the JDK `keytool` utility on the command line, for example:
 +
 [source,text,linenums]
 ----
 keytool -genkeypair -keystore keystore.jks   -dname "CN=localhost, OU=Unknown, O=Unknown, L=Unknown, ST=Unknown, C=Unknown"  -keypass password  -storepass password  -keyalg RSA  -sigalg SHA1withRSA  -keysize 2048  -alias mule  -ext SAN=DNS:localhost,IP:127.0.0.1 -validity 9999
 ----
 +
-This command creates a file named `keystore.jks`. Before proceeding, verify that this file exists and appears in the folder `src/main/resources`.
-
-. Configure an HTTPS connector inside your Mule configuration:
+You must specify the hostname (`SAN=DNS:localhost,IP:127.0.0.1` in the example) on the command line to  generate a self-signed certificate.
++
+This command creates a file named `keystore.jks`.
+. Add `keystore.jks` to your Mule app in the folder `src/main/resources` in Studio.
+. Configure an HTTP Connector in your Mule app:
```

Comment:
Should this be "Configure Anypoint Connector for HTTP..." 
or "HTTP Connector"
or "an HTTP connector"

Reply:
Anypoint Connector for HTTP is mentioned earlier. This is a subsequent mention.